backend/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "backend"
+version = "0.1.0"
+description = "The server that proves a RESTful API."
+authors = ["aashkir <ahmedaashkir@gmail.com>", "SirRam <sriram.suresh60@gmail.com>", "boarnoah <adipamihi@gmail.com>"]
+
+[tool.poetry.dependencies]
```

Comment:
I double checked and this dep list is definitely odd.

With tools like Poetry, Pipenv, Npm and other dep mgmts we only need list our direct dependencies. Its their job to handle versioning of transitive dependencies.

In our case our dependencies are:
python (just because poetry suggests listing python version explicitly like this)
Flask
Flask-SqlAlchemy
waitress

Others such as Flask-Cors, itsdangerous, jinja2, werkzeug are dependencies of Flask, not things we directly depend on.
If at some point we decide to directly use them in our own code, then we would include them as a direct dependency.

Reply:
Ok I think it should be good now.